api/tests.py

```diff
@@ -71,3 +80,274 @@ def test_options_manifest_data(self):
+        self.assertEqual(response.json()['count'],1)
+        in_goblin = json.loads(self.test_monster_json)
+        out_goblin = response.json()['results'][0]
+
+        self.assertEqual(in_goblin['name'], out_goblin['name'])
+        self.assertEqual(in_goblin['size'], out_goblin['size'])
+        self.assertEqual(in_goblin['type'], out_goblin['type'])
+        self.assertEqual(in_goblin['subtype'], out_goblin['subtype'])
+        self.assertEqual(in_goblin['group'], out_goblin['group']) #Not in original goblin
+        self.assertEqual(in_goblin['alignment'], out_goblin['alignment'])
+        self.assertEqual(in_goblin['armor_class'], out_goblin['armor_class'])
+        self.assertEqual(in_goblin['armor_desc'], out_goblin['armor_desc'])
+        self.assertEqual(in_goblin['hit_points'], out_goblin['hit_points'])
+        self.assertEqual(in_goblin['hit_dice'], out_goblin['hit_dice'])
+        self.assertEqual(in_goblin['speed_json'], out_goblin['speed']) #INPUT IS FORCED TO SPEED_JSON
+        
+        self.assertEqual(in_goblin['strength'], out_goblin['strength'])
+        self.assertEqual(in_goblin['dexterity'], out_goblin['dexterity'])
+        self.assertEqual(in_goblin['constitution'], out_goblin['constitution'])
+        self.assertEqual(in_goblin['intelligence'], out_goblin['intelligence'])
+        self.assertEqual(in_goblin['wisdom'], out_goblin['wisdom'])
+        self.assertEqual(in_goblin['charisma'], out_goblin['charisma'])
+        
+        self.assertEqual(None, out_goblin['strength_save'])
+        self.assertEqual(None, out_goblin['dexterity_save'])
+        self.assertEqual(None, out_goblin['constitution_save'])
+        self.assertEqual(None, out_goblin['intelligence_save'])
+        self.assertEqual(None, out_goblin['wisdom_save'])
+        self.assertEqual(None, out_goblin['charisma_save'])
+
+        self.assertEqual(None, out_goblin['perception'])
+        self.assertEqual(in_goblin['stealth'], out_goblin['skills']['stealth']) # MISALIGNED
+        self.assertEqual(in_goblin['damage_vulnerabilities'], out_goblin['damage_vulnerabilities'])
+        self.assertEqual(in_goblin['damage_resistances'], out_goblin['damage_resistances'])
+        self.assertEqual(in_goblin['condition_immunities'], out_goblin['condition_immunities'])
+        self.assertEqual(in_goblin['senses'], out_goblin['senses'])
+        self.assertEqual(in_goblin['languages'], out_goblin['languages'])
+        
+        self.assertEqual(in_goblin['challenge_rating'], out_goblin['challenge_rating'])
+
+        self.assertEqual(in_goblin['actions'], out_goblin['actions'])
+        self.assertEqual("", out_goblin['reactions']) #Empty string?
+        self.assertEqual("", out_goblin['legendary_desc']) #Empty string?
+        self.assertEqual("", out_goblin['legendary_actions'])#Empty string?
+        self.assertEqual(in_goblin['special_abilities'][0]['name'], out_goblin['special_abilities'][0]['name'])
```

Comment:
Could we instead assert that `in_goblin['special_abilities'] == out_goblin['special_abilities']`? Or do we have some weird different data types to deal with?